docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst

```diff
@@ -47,27 +47,30 @@ following counters are available:
   getdata_rq, Counter, Number of getdata requests
   create_rq, Counter, Number of create requests
   create2_rq, Counter, Number of create2 requests
+  createcontainer_rq, Counter, Number of createcontainer requests
+  createttl_rq, Counter, Number of createttl requests
   setdata_rq, Counter, Number of setdata requests
   getchildren_rq, Counter, Number of getchildren requests
   getchildren2_rq, Counter, Number of getchildren2 requests
-  remove_rq, Counter, Number of delete requests
+  delete_rq, Counter, Number of delete requests
```

Comment:
Consider using the same name `remove_rq` to avoid breaking it. Same goes for `remove_resp` -> `delete_resp`

Reply:
Hi @adisuissa, actually this is not a metric name change, instead, it is a document fix. Because there is no `remove_rq` or `remove_resp` specified in the code, in contrast, the ZK filter use [delete_rq](https://github.com/envoyproxy/envoy/blob/4e9c63b738d847d066166a48ca75f5669eda0da2/source/extensions/filters/network/zookeeper_proxy/filter.h#L41) and [delete_resp](https://github.com/envoyproxy/envoy/blob/4e9c63b738d847d066166a48ca75f5669eda0da2/source/extensions/filters/network/zookeeper_proxy/filter.h#L70).

If some users use the ZK filter, they will not be able to find metrics for `remove_rq` or `remove_resp`. They can only find `delete_rq` or `delete_resp`.

HTH